__tests__/app.test.js

```diff
@@ -0,0 +1,39 @@
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+const app = require("../app.js");
+const connection = require("../db/connection.js");
+
+beforeEach(() => {
+  return seed(testData);
+});
+afterAll(() => {
+  connection.end();
+});
+
+describe("GET /api/categories", () => {
+  test("GET: 200 - responds with a 200 status and an array of objects containing slug and description properties", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then(({ body }) => {
+        const { categories } = body;
+        expect(categories).toHaveLength(4);
+        expect(categories).toBeInstanceOf(Array);
+        categories.forEach((category) => {
```

Comment:
line 22 is redundant as you are using a forEach on line 23 (which will only work on an Array)